src/Xamarin.Android.Build.Tasks/Tasks/ResolveLibraryProjectImports.cs

```diff
@@ -85,7 +86,12 @@ public override bool Execute ()
 			assemblyMap.Load (AssemblyIdentityMapFile);
 
 			using (var resolver = new DirectoryAssemblyResolver (this.CreateTaskLogger (), loadDebugSymbols: false)) {
-				Extract (resolver, jars, resolvedResourceDirectories, resolvedAssetDirectories, resolvedEnvironmentFiles);
+				try {
+					Extract (resolver, jars, resolvedResourceDirectories, resolvedAssetDirectories, resolvedEnvironmentFiles);
+				} catch (ZipIOException ex) {
+					Log.LogCodedError ("XA1004", ex.Message);
+					Log.LogDebugMessage (ex.ToString ());
+				}
```

Comment:
I might be misreading this, but it looks like this error message would report the path of the temporary zip file, right?

Does it need to also report the full path of the assembly here? The issue is normally that NuGet has half downloaded an assembly, and when pulling out the zip file from EmbeddedResources we get the error?

Reply:
The files processed are in `$(IntermediateOutputPath)`, and thus are reasonably fine to show to the user.